client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -697,6 +763,28 @@ export class WorkspaceView extends React.Component<any, any> {
                           <span style={{ color: "darkGray" }}>
                             Workspace #{workspace.serialId}
                           </span>
+                          {shouldShowRootWorkspaceScratchpad && (
+                            <div
+                              style={{
+                                backgroundColor: "#f8f8f8",
+                                border: "1px solid #ddd",
+                                borderRadius: "3px",
+                                fontSize: "14px",
+                                padding: "10px",
+                                marginBottom: "10px",
+                              }}
+                            >
+                              <span style={{ fontWeight: 600 }}>
+                                Root-level scratchpad
+                              </span>
```

Comment:
Fancy! :D
![image](https://user-images.githubusercontent.com/1892071/62168959-b671bd00-b2db-11e9-93e1-94191fc70c3c.png)
Will tag @brachbach so he can confirm this design

Reply:
Discussed the UI with Ben. Concluded the minimal look is preferable (with the font color you picked). 
![image](https://user-images.githubusercontent.com/1892071/62170718-50883400-b2e1-11e9-85e3-697cbcb49a34.png)
Without the left indent on the link.

Reply:
OK I can change it to that. I personally am unsure this will look good if we ever use the root-level scratchpad to include more information (e.g., a paragraph), but we can always revisit it if that issue arises.

Reply:
That said, I do think I prefer this minimal style for something like just a link, which may be the only thing we want to use it for.

Reply:
👍